package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@snapshot-labs/snapshot.js",
-  "version": "0.5.9",
+  "version": "0.5.10",
```

Feel like this is a slightly bigger update :P
```suggestion
  "version": "0.6.0",
```